hooks/pre/30-el7_upgrade_postgresql.rb

```diff
@@ -11,6 +11,7 @@ def postgresql_12_upgrade
 
   execute!(%(scl enable rh-postgresql12 "PGSETUP_INITDB_OPTIONS='--lc-collate=#{collate} --lc-ctype=#{ctype} --locale=#{collate}' postgresql-setup --upgrade"))
   ensure_packages(['postgresql-server'], 'absent')
+  ensure_packages(['postgresql-devel'], 'absent')
```

Comment:
should we install `rh-postgresql12-postgresql-devel` to replace it? like we do with `-contrib`?